pkg/adaptor/etcd/etcd_test.go

```diff
@@ -0,0 +1,192 @@
+package etcd
+
+import (
+	"errors"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/compose/transporter/pkg/adaptor"
+	"github.com/compose/transporter/pkg/client"
+	"github.com/compose/transporter/pkg/message"
+	"github.com/compose/transporter/pkg/message/ops"
+	"github.com/compose/transporter/pkg/pipe"
+)
+
+func TestDescription(t *testing.T) {
+	e := Etcd{}
+	// confirm Etcd fulfills the Describable interface
+	e.Description()
```

Comment:
Not sure this is sort of test is useful, we already do a compile time check for `Describable` [here](https://github.com/compose/transporter/pull/255/files#diff-6543c64e145ee13ebabcdca9c8ec884fR16)